e2e-tests/portal/cypress/e2e/journeys/maker/amend-facility.spec.js

```diff
@@ -0,0 +1,139 @@
+const MOCK_USERS = require('../../../fixtures/users');
```

Comment:
At some point we should think about the case of not showing amendment buttons when an amendment has started, although I imagine that first happens when we start to save data (on the next screen) and change the state of the facility. Not something to test here but need to remember it - is it worth adding a TODO with a ticket number I'll find in a bit to make sure we add tests for when the buttons shouldn't show? I guess same for GEF!